furusiya/model/skills/omnislash.py

```diff
@@ -0,0 +1,9 @@
+import random
+
+
+class OmniSlash:
+    @classmethod
```

Comment:
Static please. Doesn't depend on `cls`. (That's the "rule of thumb" I learned :))